hawkbit-runtime/hawkbit-update-server/src/test/java/org/eclipse/hawkbit/app/CorsTest.java

```diff
@@ -38,8 +39,9 @@
 @RunWith(SpringRunner.class)
 @SpringBootTest(properties = {"hawkbit.dmf.rabbitmq.enabled=false", "hawkbit.server.security.cors.enabled=true",
         "hawkbit.server.security.cors.allowedOrigins=" + CorsTest.ALLOWED_ORIGIN_FIRST + "," + CorsTest.ALLOWED_ORIGIN_SECOND})
-@TestExecutionListeners(listeners = { MySqlTestDatabase.class, MsSqlTestDatabase.class }, 
+@TestExecutionListeners(listeners = { MySqlTestDatabase.class, MsSqlTestDatabase.class },
         mergeMode = MergeMode.MERGE_WITH_DEFAULTS)
+@DirtiesContext
```

Comment:
Test works fine without `@DirtiesContext`. Can it be removed?

Reply:
Thanks for the hint, removed it.